uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/instances/leave-request.instance.js

```diff
@@ -64,6 +96,25 @@ define([
         return $q.all(promises);
       }
 
+      /**
+       * Gets time deduction selected by user in case it is a working day
+       * or returns the calculated deduction in case it is not a working day
+       *
+       * @param  {Object} day taken from a balance change breakdown
+       * @param  {String} type (from|to)
+       * @param  {Object} dayTypes as per OptionGroup
+       * @return {Number}
+       */
+      function getDayBalanceChangeForHoursUnit (request, day, type, dayTypes) {
+        var dayType = _.find(dayTypes, { value: String(day.type.value) }).name;
+
+        if (!_.includes(['weekend', 'non_working_day', 'public_holiday'], dayType)) {
```

Comment:
You can use a ternary operator here.

Reply:
Not relevant anymore since after a call with @AkA84 we decided this code is not needed

Reply:
Please see above, it is not relevant anymore.